CohortsScheduling/src/main/java/dataModels/StartRequest.java

```diff
@@ -0,0 +1,26 @@
+package dataModels;
+import java.util.List;
+import org.springframework.web.multipart.*;
+public class StartRequest {
+	private MultipartFile file;
+	private List<Requirement> requirements;
+	private int count;
+	
```

Comment:
need schedule name and to add it as a field in each cohort schedule. we can do that later though